.vscode/launch.json

```
@@ -0,0 +1,18 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```

Technically JSON doesn't support comments
```suggestion
    "__comment": "Use IntelliSense to learn about possible attributes."
```